src/NUnitConsole/nunit3-console/nunit3-console.csproj

```diff
@@ -9,6 +9,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Commandlineparameters>nunit.engine.tests.dll -process:Single</Commandlineparameters>
     <LangVersion>7</LangVersion>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
```

Comment:
We could think about putting these repeated elements in `src/Directory.Build.props`.

Reply:
Strange coincidence. I was thinking the same during a long run today, but I could not remember whether we had such a file in this repo, and then I forgot all about it again :).